css-four-column-layout/styles.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Barlow, sans-serif;
+  font-size: 0.7rem;
+}
+
+h1,
+h4 {
+  font-family: Barlow, sans-serif;
+}
+
+.how-it-works {
+  background-color: #f7f8fa;
+  text-align: center;
+}
+
+.container {
+  max-width: 1130px;
+  margin: 0 auto;
+}
+
+.max-width-shrink {
+  max-width: 700px;
+}
```

Comment:
You should just make the `max-width` on the `container` class `700px` if that is the measurement you want it to have 🤓

Reply:
got it!